turbinia/workers/be.py

```diff
@@ -49,8 +50,7 @@ def run(src_path, out_path, offsets, job_id, **kwargs):
 class BulkExtractorCalcOffsetsTask(TurbiniaTask):
   """Task to calculate offsets for Bulk extractor."""
 
-  @staticmethod
-  def run(src_path, num_workers, page_size=16777216):
+  def run(self, evidence, num_workers, page_size=16777216):
```

Comment:
(I'm assuming that is what you meant.  Let me know if you meant something else).